common/network/network.go

```diff
@@ -0,0 +1,69 @@
+package network
+
+import (
+	"fmt"
+	"math/big"
+	"slices"
+
+	"github.com/scroll-tech/go-ethereum/params"
+)
+
+// Network represents a known network
+type Network string
+
+var (
+	Mainnet Network = "mainnet"
+	Sepolia Network = "sepolia"
+	Alpha   Network = "alpha"
+)
+
+// IsKnown returns if the network is indeed known
+func (n Network) IsKnown() bool {
+	return n == Mainnet || n == Sepolia || n == Alpha
+}
+
+// GenesisConfig returns the chain config of a known network
+func (n Network) GenesisConfig() *params.ChainConfig {
+	switch n {
+	case Mainnet:
+		return params.ScrollMainnetChainConfig
+	case Sepolia:
+		return params.ScrollSepoliaChainConfig
+	case Alpha:
+		return params.ScrollAlphaChainConfig
+	default:
+		panic(fmt.Sprintf("unknown network (%s), check configuration", n))
+	}
+}
+
+// CollectSortedForkHeights returns a sorted set of block numbers that one or more forks are activated on
+func CollectSortedForkHeights(config *params.ChainConfig) []uint64 {
+	var forkHeights []uint64
+	for _, fork := range []*big.Int{
+		config.HomesteadBlock,
+		config.DAOForkBlock,
+		config.EIP150Block,
+		config.EIP155Block,
+		config.EIP158Block,
+		config.ByzantiumBlock,
+		config.ConstantinopleBlock,
+		config.PetersburgBlock,
+		config.IstanbulBlock,
+		config.MuirGlacierBlock,
+		config.BerlinBlock,
+		config.LondonBlock,
+		config.ArrowGlacierBlock,
+		config.ArchimedesBlock,
+		config.ShanghaiBlock,
+	} {
+		if fork == nil {
+			continue
+		} else if !fork.IsUint64() {
+			panic("fork set to a height larger than max uint64")
+		} else if height := fork.Uint64(); height != 0 {
+			forkHeights = append(forkHeights, height)
+		}
```

Comment:
```suggestion
		forkHeights = append(forkHeights, fork.Uint64())
```

How about change to this. because the `[]*big.Int` is decided, so the exception check is redundancy, i think

Reply:
Sure, if you say it is safe to assume fork heights to always fit in a `uint64`.